README.adoc

```diff
@@ -144,58 +142,44 @@ Create the `InventoryManager` class in the
-number of currently executing invocations of the annotated method. In other words, it makes the counter increment
-when the annotated method is called and decrement when it returns.
+| `monotonic` | Set to `true` to count the total number of invocations of the annotated method i.e.
+the total number of accesses.
 |===
 
-The `@Gauge` annotation on the `getTotal()` method tracks the number of systems that are stored in
-the inventory. A system is stored in the inventory when you visit the
-`http://localhost:9080/inventory/systems/localhost` URL to retrieve the information for a specific host.
-See the additional metadata field:
+Apply the `@Gauge` annotation to the `getTotal()` method to track the number of systems that are stored in
+the inventory. Note that when the value of the gauge is retrieved, the underlying `getTotal()` method
+is called to return the size of the inventory.
 
-|===
-| `unit` | Sets the unit of the metric
-|===
-
-Note that, when the value of the gauge is retrieved, the underlying `getTotal()` annotated method is
-called to return the value.
-
-These annotations are registered by default to the `MetricRegistry` singleton object.
-This object is queried for metadata and metrics when you call the `https://localhost:9443/metrics/application`
-MicroProfile Metrics endpoint, which is provided for you. Also, the `@Counted`, `@Gauge`, and `@Timed` annotations can be
-used in services, such as the `inventory` service in the
-`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file.
+Additional information about these annotations, relevant metadata fields and more is available at
+the https://microprofile.io/project/eclipse/microprofile-metrics[MicroProfile website].
 
 // =================================================================================================
 // Building and running the application
 // =================================================================================================
 
 include::{common-includes}/mvnbuild.adoc[]
 
-Point your browser to the `https://localhost:9443/metrics` URL to find the metrics that show JVM
-and server base details.
+Point your browser to the `https://localhost:9443/metrics` URL to review the all available metrics
+that has been enabled through MicroProfile Metrics. Login with `confAdmin` and `microprofile`. You
+see only the system metrics because the server just started and the `inventory` service has not been
+accessed.
 
-Next, point your browser to `http://localhost:9080/inventory/systems`. Then, you can find
-specific metrics about your application at the `https://localhost:9443/metrics/application` URL.
+Next, point your browser to the `http://localhost:9080/inventory/systems` URL. Reload
+`https://localhost:9443/metrics` or you can access the application metrics only with the URL, 
+`https://localhost:9443/metrics/application`.
```

Comment:
ID changes:
"Reload the `https://localhost:9443/metrics` URL, or access only the application metrics at the `https://localhost:9443/metrics/application` URL."